lib/uptime/storage.ex

```diff
@@ -68,4 +69,16 @@ defmodule Uptime.Storage do
   end
 
   defp maybe_preload_checks(query, _preload_checks), do: query
+
+  @spec broadcast!(Check.t()) :: :ok
```

Comment:
this should move up a level to Uptime, storage is solely for the functions that touch the storage layer, all of the other business logic lives in the api context (although Uptime can call out to other modules such as http clients that are in its context)

Reply:
Oh okay I see now